src/hotspot/cpu/s390/register_s390.hpp

```diff
@@ -1,6 +1,6 @@
 /*
- * Copyright (c) 2016, 2019, Oracle and/or its affiliates. All rights reserved.
- * Copyright (c) 2016, 2019 SAP SE. All rights reserved.
+ * Copyright (c) 2016, 2023, Oracle and/or its affiliates. All rights reserved.
+ * Copyright (c) 2016, 2023 SAP SE. All rights reserved.
```

Comment:
Hi All, is it okay If I add IBM header as well ?

Reply:
Thanks Lutz, Let's skip this part then :-)

Reply:
I agree. You can add IBM Copyright headers when you contribute new files or when they contain substantial contributions.

Reply:
My thoughts were that we could update header once we do substantials changes in the file, I haven't considered the port as whole. But I'll take care of it next time.